Number_types/include/CGAL/Quotient.h

```diff
@@ -565,7 +565,7 @@ namespace INTERN_QUOTIENT {
         public:
           NT operator()( const NT& x ) const {
             CGAL_precondition(x > 0);
-            return NT(CGAL_NTS sqrt(x.numerator()*x.denominator()),
+            return NT(CGAL_NTS sqrt(typename NT::NT(x.numerator()*x.denominator())),
```

Comment:
`typename NT::NT` -> just `NT` (or `static_cast<NT>`) if that's what you mean.
Is it just because of https://github.com/boostorg/multiprecision/issues/319 (and it could be guarded by a macro) or was it motivated by an other issue?

Reply:
`NT = Quotient` AFAIU, here we want `sqrt` to be applied on the integer type, that is `NT::NT`. Looks like we were affected by  the issue you reported.

Reply:
I see, sorry for the wrong comment. Using the same name `NT` is quite misleading...